src/main/java/ch/njol/skript/expressions/ExprCommandInfo.java

```diff
@@ -43,108 +50,106 @@
 
 @Name("Command Info")
 @Description("Get information about a command.")
-@Examples({"main name of command \"skript\"",
+@Examples({
+	"main name of command \"skript\"",
```

Comment:
This isn't a valid example, should probably be changed while you're here. ``main command label of command "skript"`` is, though. Also see other comment about mismatched syntax.